packages/desktop-client/src/components/payees/PayeeMenu.tsx

```diff
@@ -32,6 +32,11 @@ export function PayeeMenu({
     id => payeesById[id] == null || payeesById[id].transfer_acct,
   );
 
+  const selectedPayeeNames = [...selectedPayees]
+    .slice(0, 4)
+    .map(id => payeesById[id].name)
+    .join(', ');
```

Comment:
One more quick suggestion: we have a `useFormatList` hook that is perfect for this type of thing! Maybe we can even adapt that hook to take an optional max # and auto-abbreviate with "and more" once the limit is reached